src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,40 @@
+package com.google.sps.data;
+
+import java.util.Date;
+
+public class Comment {
+
+  private final String author;
+  private final String messageContent;
+  private final Date timeStamp;
```

Comment:
**nit**

timestamp is one word so camel case is not necessary.

Reply:
This I did not know. Thanks!